### DISCUSSION TITLE
Related posts section says "[MISSING TRANSLATION]" instead of something like "Related Posts"

### DISCUSSION THREAD

**Discussion by jjets718** [GetPublii/Publii#1670] [Help] [answered]
Hi there! I'm using the **[Sources theme](https://marketplace.getpublii.com/themes/sources/)**, and right above the related posts section, I noticed the h5 says "[MISSING TRANSLATION]" instead of something like "Related Posts". Has anyone encountered this issue, and if so, do you have a fix? Please let me know.

**Comment by bobmitro** [comment]
I don't see such a problem in the Source theme; it may result from some of your modifications. 
You have to provide the missing phrase in the language file; that's it.

**Reply by jjets718** [reply]
Hi Bob - thanks for your note. It was working and then it wasn't. I've always had "Related posts" in the language file. I just updated it  to use different copy and reinstalled the theme with the updated file (zipped everything and added it as a new version of the theme). This new copy isn't showing up either.

I looked in the post.hbs file and this code is in there too:

<h3 class="h5">
                  {{ translate 'post.relatedPosts' }}
</h3>

So, I don't understand what the issue is.

**Reply by jjets718** [reply, marked as answer]
Bob, I figured it out. I had to change the name of the sources.lang.json file to the new name of my theme (to mark a few updates I made to it), and that fixed the issue. I ended up finding your reply in a different thread. Thanks!
